src/main/java/org/asciidoctor/maven/AsciidoctorMojo.java

```diff
@@ -242,10 +319,9 @@ private void setDestinationPaths(OptionsBuilder optionsBuilder, final File sourc
 
     protected Asciidoctor getAsciidoctorInstance(String gemPath) throws MojoExecutionException {
         Asciidoctor asciidoctor = null;
-        if (gemPath == null) {
-             asciidoctor = Asciidoctor.Factory.create();
-        }
-        else {
+        if (gemPath == null || gemPath.isEmpty()) {
```

Comment:
I totally believe you but I want to understand this. Wasn't `null` the trick to really isolate it? I recall this has been changed in 1.6.0 to inject a null when using 'create()'.
So, what's the difference between create(), create(null) and create("")?